src/mobile/components/layout/test/layout.jest.ts

```diff
@@ -14,6 +14,7 @@ describe("bootstrap", () => {
   })
 
   it("sets up mediator to call #logoutEventHandler", () => {
+    // @ts-ignore
```

Comment:
why is this suddenly needed?

Reply:
i was trying to resolve this error (only happens in [CI](https://app.circleci.com/pipelines/github/artsy/force/22348/workflows/ac01593a-b5ea-428e-b91f-22ecdfbcaf40/jobs/170014)):
```
FAIL   legacy  src/mobile/components/layout/test/layout.jest.ts (568 MB heap size)
  ● Console

    console.log src/lib/analytics/segmentOneTrustIntegration/segmentOneTrustIntegration.ts:17
      loadSegment called

  ● bootstrap › sets up mediator to call #logoutEventHandler

    TypeError: Cannot read property 'load' of undefined

      34 |     // OneTrust is disabled. Load Segment normally - load all Segment destinations.
      35 |     // @ts-ignore
    > 36 |     window.analytics.load(SEGMENT_WRITE_KEY)
         |                      ^
```
but it didn't help.

Reply:
this was resolved by https://github.com/artsy/force/pull/8853/commits/fdc74ca09fefb92840e63b93625023b542ed022a